css/portal-dashboard/student-list.less

```diff
@@ -3,4 +3,6 @@
 .studentList {
   width: 100%;
   background-color: white;
+  overflow: auto;
+  height: calc(100vh - 245px - 60px);
```

Comment:
Seems like a good place for some variables rather than magic numbers.

Reply:
Yeah, I was waiting for those components to settle down, but now is as good of a time as any to slot in some variables.

Reply:
Fixed.  Thanks!